plugins/backend/catalog-backend-module-okta/README.md

```diff
@@ -125,9 +125,12 @@ export default async function createPlugin(
 
   const orgProvider = OktaOrgEntityProvider.fromConfig(env.config, {
     logger: env.logger,
-    parentGroupField: 'parent_org_id',
     userNamingStrategy: 'strip-domain-email',
     groupNamingStrategy: 'kebab-case-name',
+    hierarchyConfig: {
+      key: 'profile.org_id',
```

Comment:
These could these be named `orgId` and `parentOrgId` instead for clarity

Reply:
done.